### PR TITLE
Feature/256 explore making webform available offline

### PR DESCRIPTION
**Pull request by wayangalihpratama**
- #256

### CODE REVIEW

File: backend/tests/test_17_webform_editor.py [hunk 1]
```diff
@@ -794,7 +794,7 @@ async def test_get_all_form(self, app: FastAPI, session: Session,
                 "language": "id",
                 "description": "Ini adalah formulir uji coba untuk webform",
```
Comment (dedenbangkit) [inline backend/tests/test_17_webform_editor.py:795]: Please use English or LipSum for if you want to write any sentence @wayangalihpratama

File: frontend/src/serviceWorkerDev.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    const url = `${process.env.PUBLIC_URL}/serviceWorker.js`;
+    navigator.serviceWorker
+      .register(url, {
+        scope: '/webform',
+      })
+      .then((registration) => {
+        if (registration.installing) {
+          console.info('Service worker installing');
+        } else if (registration.waiting) {
+          console.info('Service worker installed');
```
Comment (dedenbangkit) [inline frontend/src/serviceWorkerDev.js:12]: How we uninstall the service worker / do we have any trigger to reinstall the service worker (e.g check the website version) ?

**Review by dedenbangkit — CHANGES_REQUESTED** [review]

File: frontend/src/serviceWorkerDev.js (hunk 2 above)
Comment (wayangalihpratama) [reply frontend/src/serviceWorkerDev.js:12]: For now, we don't have a trigger for that. I will check for this and update this when I work on issue #274 .

File: backend/tests/test_17_webform_editor.py (hunk 1 above)
Comment (wayangalihpratama) [reply backend/tests/test_17_webform_editor.py]: I have updated this one.

**Review by dedenbangkit — APPROVED** [review]

**Pull request merged**